### PR TITLE
add rating

### PR DESCRIPTION
**Pull request by harshkotkar5557**
rating_kaizen-ui

### CODE REVIEW

File: rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.rating-box{
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (vaibhav18matere) [inline rating/rating.css:4]: can make separate class for this

**Review by vaibhav18matere — APPROVED** [review]

File: rating/rating.css (hunk 1 above)
Comment (harshkotkar5557) [reply rating/rating.css:4]: okay i will do that

**Review by omgaikwad — APPROVED** [review]
Code looks good, you can merge.

**Review by aamrapaliwandhre — APPROVED** [review]

**Comment by aamrapaliwandhre** [comment]
Code looks good to me....go ahead

**Pull request merged**